pcli/src/legacy.rs

```diff
@@ -44,8 +44,8 @@ pub struct ClientState {
 
 /// A legacy wallet file.
 #[derive(Debug, Clone, Serialize, Deserialize)]
-#[serde(from = "serde_helpers::WalletHelper")]
-#[serde(into = "serde_helpers::WalletHelper")]
+#[serde(from = "serde_helpers::WalletHelper")] // Deserialize from legacy format
+#[serde(into = "penumbra_custody::soft_kms::Config")] // Serialize into non-legacy format
```

Comment:
I think we don't want to do this just yet, because we haven't figured out how we want to structure `pcli` configs going forward (where there might be multiple custody providers); if we stick with the existing structure rather than using the new `Config`, we can keep it unchanged until we know what we want the new format to be.

Reply:
I think we actually don't want to be touching this at all, this isn't the part of the `pcli` code that's currently used to store the key material.